### ISSUE TITLE
loss overflow during training..?

### ISSUE THREAD

**Issue by geonuk-kimmm** [chanyn/HKRM#18]
Hi, i am gonna train HKRM, but i got error about loss.
when i train on default hyper-parameters, following error occurs. 
![image](https://user-images.githubusercontent.com/34879386/62845770-c50d9a80-bd05-11e9-937d-5b07874ae68b.png)
It seems to error shows too big for start training. So i changed (init) learning rate to 0.0000000000001 and training starts well. But It is too slow to be optimized.
I'd like to ask about the error, what occurs the error and how could i fix it in the code?
Thx

**Comment by guyuchao** [comment]
> Hi, i am gonna train HKRM, but i got error about loss.
> when i train on default hyper-parameters, following error occurs.
> ![image](https://user-images.githubusercontent.com/34879386/62845770-c50d9a80-bd05-11e9-937d-5b07874ae68b.png)
> It seems to error shows too big for start training. So i changed (init) learning rate to 0.0000000000001 and training starts well. But It is too slow to be optimized.
> I'd like to ask about the error, what occurs the error and how could i fix it in the code?
> Thx

What and how many GPU do you use?

**Comment by geonuk-kimmm** [comment]
Thx for reply. I use two TITAN Xp!

**Comment by guyuchao** [comment]
> Thx for reply. I use two TITAN Xp!

Have you encountered out-of-memory in calculating Adjacent matrix？

`W_new = torch.abs(W1 - W2)`
This step will produce a batchsize*128*128*2048 matrix, then causes OOM error.

**Comment by Lv-shuai** [comment]
> 
bro，How did you solve this problem？I met the same situation
